client/src/components/manageProjects/editProject.js

```diff
@@ -207,4 +198,21 @@ const EditProject = ({
   );
 };
 
+function RecurringEvent({event, setSelectedEvent}) {
+
+  return (
+    <li key={`${event.event_id}`}>
+      <button type="button" onClick={() => {console.log(event);setSelectedEvent(event)}}>
```

Comment:
`setSelectedEvent` designates which event should be editable in the modal rendered by `<EditMeetingTimes />`